packages/@react-native-windows/cli/src/test/dependencyConfig.test.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+
+import {
+  dependencyConfigWindows,
+  WindowsDependencyConfig,
+} from '../config/dependencyConfig';
+
+import {copyAndReplace} from '../generator-common';
+
+const templateRoot = path.resolve('../../../vnext/template');
+
+const testProjectGuid = '{416476D5-974A-4EE2-8145-4E331297247E}';
+
+function projectTests(testLibName: string, setup?: Function): void {
```

Comment:
nit: `Function` doesn't have great type safety, and some linters will tell you not to use it (we don't have the rules enabled right now). Consider something like `() => void` instead, or `() => Promise<void>` if async.

Reply:
Fixed.